### DISCUSSION TITLE
How do I delete an organization that I created from converting from a user

### DISCUSSION THREAD

**Discussion by aHumes** [community/community#40385] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello all I don't know if this is a bug but it sure feels like it. My company was trying to create a git hub repo for the company. We created a user with our company name. We then converted that user into an organization. We can no longer login to that user (organization rules) but we also don't own that organization from any other user. It looks like it just ate our company name and is out there in the ether floating around. Any suggestions would be most helpful. Appreciate it!

**Comment by radar-blip** [comment, marked as answer]
Sounds like something you may have to contact support about. Here is a link for that: [Support Page](https://support.github.com/request)
Maybe under `Account or billing issues` -or- `Remove data from a repository I own or control`
